src/virtual_dom/node.rs

```diff
@@ -19,7 +19,7 @@ pub enum Node<Ms: 'static> {
     Empty,
 }
 
-impl<Ms: 'static> Clone for Node<Ms> {
+impl<Ms: Clone + 'static> Clone for Node<Ms> {
```

Comment:
I hoped we can remove all `Clone` restrictions for `Ms` because it force us to do some ugly things like wrapping `Msg` variants to `Rc` in some applications.

Reply:
This is needed to clone `Listener<Ms>` who owns `message: Option<Ms>`.

We can remove `message: Option<Ms>` this way we can clone `Listener<Ms>` thus we can clone `Node<Ms>` without restricting `Ms: Clone.`.

if I understand correctly `message: Option<Ms>` is only used in `impl<Ms> PartialEq for Listener<Ms>` which I think it isn't important and it isn't complete too.

Reply:
> if I understand correctly message: Option<Ms> is only used in impl<Ms> PartialEq for Listener<Ms> which I think it isn't important and it isn't complete too.

It will be probably a part of my VDOM refactor so it's possible I'll resolve it. So we can leave it as it is for now.